You can remove the import of the job function
```
sms_939/models/recurring_contract.py:12:1: F401 'odoo.addons.queue_job.job.job' imported but unused
```

--- a/sms_939/models/recurring_contract.py
+++ b/sms_939/models/recurring_contract.py
@@ -9,7 +9,6 @@
 #
 ##############################################################################
 from odoo import api, models
-from odoo.addons.queue_job.job import job
 
 
 class Contract(models.Model):
@@ -36,21 +35,4 @@ class Contract(models.Model):
     def associate_group(self, payment_mode_id):
         res = super(Contract, self).associate_group(payment_mode_id)
         self.group_id.on_change_payment_mode()
-        return res
-
-    @api.model
-    @job
-    def create_sms_sponsorship(self, vals, partner, sms_child_request):
-        """ Creates sponsorship from REACT webapp data.
-        - Mark the partner in pending state if it is a new partner.
-
-        :param vals: form values
-        :param partner: res.partner record
-        :param sms_child_request: sms.child.request record
-        :return: True
-        """
-        res = super(Contract, self).create_sms_sponsorship(
-            vals, partner, sms_child_request)
-        if sms_child_request.new_partner:
-            sms_child_request.partner_id.state = 'pending'
         return res

